pkgs/applications/misc/signumone-ks/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchurl, dpkg, autoPatchelfHook, makeWrapper,
+  atk, ffmpeg, gdk-pixbuf, glibc, gtk3, libav_0_8, libXtst }:
+
+stdenv.mkDerivation rec {
+  pname = "signumone-ks";
+  version = "3.1.2";
+
+  src = fetchurl {
+    url = "https://cdn-dist.signum.one/${version}/${pname}-${version}.deb";
+    sha256 = "4efd80e61619ccf26df1292194fcec68eb14d77dfcf0a1a673da4cf5bf41f4b7";
+  };
+
+  # Necessary to avoid using multiple ffmpeg and gtk libs
+  autoPatchelfIgnoreMissingDeps = true;
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeWrapper
+  ];
+
+  buildInputs = [
+    atk glibc gdk-pixbuf stdenv.cc.cc ffmpeg
+    libav_0_8 gtk3 libXtst
```

Comment:
I doubt we need both `ffmpeg` and `libav_0_8`.
Also, shouldn't `glibc` and `stdenv.cc.cc` automatically be added by `stdenv.mkDerivation`?

Reply:
Not adding libav_9_8 will not build.

```
autoPatchelfHook could not satisfy dependency libavcodec-ffmpeg.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-56.so
autoPatchelfHook could not satisfy dependency libavcodec.so.53 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-53.so
autoPatchelfHook could not satisfy dependency libavcodec.so.57 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-57.so
autoPatchelfHook could not satisfy dependency libavcodec.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-56.so
autoPatchelfHook could not satisfy dependency libavcodec.so.55 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-55.so
autoPatchelfHook could not satisfy dependency libavcodec.so.54 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-54.so
autoPatchelfHook could not satisfy dependency libavformat-ffmpeg.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-56.so
autoPatchelfHook could not satisfy dependency libavformat.so.54 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-54.so
autoPatchelfHook could not satisfy dependency libavformat.so.55 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-55.so
autoPatchelfHook could not satisfy dependency libavformat.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-56.so
autoPatchelfHook could not satisfy dependency libavformat.so.57 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-57.so
autoPatchelfHook could not satisfy dependency libavformat.so.53 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-53.so
/nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3
```

Reply:
Never mind, I forgot that it is ignoring missing deps. It is running as expected. Going to fix it.

Reply:
> Not adding `libav_0_8` will not build.
> 
> ```
> autoPatchelfHook could not satisfy dependency libavcodec-ffmpeg.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-56.so
> autoPatchelfHook could not satisfy dependency libavcodec.so.53 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-53.so
> autoPatchelfHook could not satisfy dependency libavcodec.so.57 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-57.so
> autoPatchelfHook could not satisfy dependency libavcodec.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-56.so
> autoPatchelfHook could not satisfy dependency libavcodec.so.55 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-55.so
> autoPatchelfHook could not satisfy dependency libavcodec.so.54 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-54.so
> autoPatchelfHook could not satisfy dependency libavformat-ffmpeg.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-56.so
> autoPatchelfHook could not satisfy dependency libavformat.so.54 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-54.so
> autoPatchelfHook could not satisfy dependency libavformat.so.55 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-55.so
> autoPatchelfHook could not satisfy dependency libavformat.so.56 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-56.so
> autoPatchelfHook could not satisfy dependency libavformat.so.57 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-ffmpeg-57.so
> autoPatchelfHook could not satisfy dependency libavformat.so.53 wanted by /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3/opt/SignumOne-KS/runtime/lib/amd64/libavplugin-53.so
> /nix/store/zzzb60wwkcyq71n1h8gk6idjx66djzd6-signumone-ks-3.1.3
> ```

This probably means it requires `libav_0_8`, but not `ffmpeg`.

Reply:
Actually, you should be fine with ffmpeg_3:
```
$ nix-build -A ffmpeg_3
$ ls result/lib
libavcodec.so             libavfilter.so.6          libavresample.so.3.7.0   libswresample.so
libavcodec.so.57          libavfilter.so.6.107.100  libavutil.so             libswresample.so.2
libavcodec.so.57.107.100  libavformat.so            libavutil.so.55          libswresample.so.2.9.100
libavdevice.so            libavformat.so.57         libavutil.so.55.78.100   libswscale.so
libavdevice.so.57         libavformat.so.57.83.100  libpostproc.so           libswscale.so.4
libavdevice.so.57.10.100  libavresample.so          libpostproc.so.54        libswscale.so.4.8.100
libavfilter.so            libavresample.so.3        libpostproc.so.54.7.100
```